<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server</class>
 <widget class="QWidget" name="server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>server</string>
  </property>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2Nokta">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>20</y>
     <width>16</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBaglan">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IP-Port Onayı</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDurum">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(212, 0, 0);</string>
   </property>
   <property name="text">
    <string>Server Başlamadı....</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>171213082 Yasir Köse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGelen">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
